packages/zoe/test/unitTests/bounty.js

```diff
@@ -48,7 +49,7 @@ const start = async zcf => {
       zcf.shutdown('The bounty was not earned');
     }
 
-    /** @type {OfferHandler} */
+    /** @type {(bountSeat: ZCFSeat) => void} */
```

Comment:
```suggestion
    /** @type {(bountySeat: ZCFSeat) => void} */
```

Reply:
Oh, yeah.  That caught my eye at one point, but I didn't hold onto it long enough to actually fix it.  

`lint` didn't point it out; probably because it's in a test.